### DISCUSSION TITLE
Use nemo as file picker to solve thumbnail bug

### DISCUSSION THREAD

**Discussion by vadimk1337** [linuxmint/discussions#623] [Ideas]
Just like they did in Ubuntu 24.10:
https://www.omgubuntu.co.uk/2024/09/ubuntu-24-10-fixes-missing-thumbnails-file-picker

**Comment by vadimk1337** [comment]
Now gnome, kde plasma, cosmic generate icons in the picker file, but cinnamon does not.

**Comment by Qronikarz** [comment]
Related nemo feature request - https://github.com/linuxmint/nemo/issues/3068
